README.md

```diff
@@ -1,58 +1,28 @@
 # Toggl Button Chrome extension
 
-Add Toggl one-click time tracking to popular web tools.
+Add Toggl one-click time tracking to Trello.
 
 ## Compatible services
```

Comment:
You can say that the other services will potentially be implemented in the future